### ISSUE TITLE
Build Fail 'Could not compile `bitflags`'

### ISSUE THREAD

**Issue by Canalytic** [energywebfoundation/energyweb-client#22]
Am trying to build the client on Ubuntu 16.

I have a Parity client already installed.  So question prior to debugging this error is: Why can't I just run parity with the --chain=Tobala flag ??

`/energyweb-client$ cargo build 
warning: unused manifest key: patch
   Compiling siphasher v0.1.3
   Compiling protobuf v1.5.1
   Compiling bitflags v0.9.1
   Compiling macros v0.1.0 (file:///home/m/energyweb-client/util/macros)
   Compiling unicode-width v0.1.4
   Compiling interleaved-ordered v0.1.1
   Compiling bitflags v1.0.1
   Compiling fnv v1.0.5
error: expected ident, found #
   --> /home/m/.cargo/registry/src/github.com-1ecc6299db9ec823/bitflags-1.0.1/src/lib.rs:423:29
    |
423 |                             #[allow(deprecated)]
    |                             ^
    | 
   ::: /home/m/.cargo/registry/src/github.com-1ecc6299db9ec823/bitflags-1.0.1/src/example_generated.rs
    |
4   | bitflags! {
    | - in this macro invocation

error: Could not compile `bitflags`.
Build failed, waiting for other jobs to finish...
error: build failed`

**Comment by steffenkux** [comment]
--chain=Tobalaba will be included in future.
To your compile problems: this looks like your Rust version is not up to date. You have to update to the current version. Then it will compile.
For compiling in more detail, please refer directly to parity.

**Comment by Canalytic** [comment]
I still could never get a clean compile.  I checked and am running most up-to-date version of Rust.  I already have a stable build of parity, and so what I ended up doing was to search down the chainspec configuration here: https://github.com/energywebfoundation/energyweb-client/blob/master/ethcore/res/ethereum/tobalaba.json

I don't see why this isn't made more explicit in the readme, because it seems much easier to get stable build of parity and then run with this chainspec.
